beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -230,16 +232,9 @@ func (vs *Server) buildPhase0BlockData(ctx context.Context, req *ethpb.BlockRequ
 		return nil, fmt.Errorf("could not get ETH1 data: %v", err)
 	}
 
-	// Pack ETH1 deposits which have not been included in the beacon chain.
-	deposits, err := vs.deposits(ctx, head, eth1Data)
+	deposits, atts, err := vs.depositAndPackAttestations(ctx, head, eth1Data)
```

Comment:
it is a bit odd to group deposit and attestation packing into one method. But I do understand that it makes it cleaner when applying the flag.